### ISSUE TITLE
Getting "no such file or directory, rename ..."

### ISSUE THREAD

**Issue by mcaoun** [loopline-systems/electron-builder#208]
Since version 2.8.4 I get the error

    No such file or directory, rename 'C:\projects\myapp\desktop\dist\win\Setup.exe' -> 'C:\projects\myapp\desktop\dist\win\MyAppSetup-1.1.0.exe

I have tested both on my Windows machine and on Appveyor and the same error occurs.

It doesn't happen if I revert back to 2.8.3

I also see a new directory being created at the same location as my package.json, in my case `C:\projects\myapp\desktop`: `C:\projects\myapp\desktop\projects\myapp\desktop`, in that directory I can find the missing Setup.exe which rename could not find.

I assume some confusion between relative and absolute paths somehow..?

**Comment by develar** [comment]
Yes, winstaller is broken. I have set latest to 2.8.3.

**Comment by mcaoun** [comment]
I cannot benefit from the fix to https://github.com/loopline-systems/electron-builder/issues/190 until this one is resolved. Correct?

**Comment by develar** [comment]
@mcaoun Yes, blocked on. I am going to publish pre-release version in a few hours.

**Comment by develar** [comment]
@mcaoun Please try version `next`.

**Comment by mcaoun** [comment]
yes, it looks good as far as I can tell, thanks

**Comment by develar** [comment]
@mcaoun #223 prevents you to use RELEASES files, I am working on fix.

**Comment by uggrock** [comment]
I am getting this issue in my windows build with 3.1.0 and 3.0.0

**Comment by develar** [comment]
@uggrock Please file new issue and provide full error log.
